lib/pages/trending/widgets/info_card.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:soundspace/constants/style.dart';
+import 'package:soundspace/constants/music_list.dart';
+import 'package:soundspace/pages/trending/widgets/components/custom_list_tile.dart';
+
+class InfoCard extends StatelessWidget {
```

Comment:
currently you are having the child track its own state using Inkwell if I understand correctly. This will not work moving forward as state tracking has a large overhead so it will be more efficient later to just have InfoCard be a Statefull widget and define animations under it.